### DISCUSSION TITLE
Deployment on Debian VPS

### DISCUSSION THREAD

**Discussion by DzNab** [joemasilotti/railsdevs.com#897] [Q&A]
Hi Joe,

First of all, thank you for the work you did on developing this project & making it available open source.

In my journey of trying to implement it for my project, I am facing hard time deploying in a Debian VPS... even if deploying it on my mac was possible and straightforward for me.

As a beginner, I find it quite difficult to figure out what are the steps to do the deployment, so I wanted to know if you have any detailed tutorial that explains it?

Thank you very much for your help,

Nabil

**Comment by joemasilotti** [comment]
I've only deployed this directly to Heroku. So sadly I won't be able to help if you are trying to deploy it anywhere else. But do check the [production settings documentation](https://github.com/joemasilotti/railsdevs.com/blob/main/docs/production_settings.md) for information on secrets and credentials.
